examples/flava/tools/convert_weights.py

```diff
@@ -0,0 +1,79 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+# # Copyright (c) Meta Platforms, Inc. and affiliates.
```

Comment:
Is the second license header added by the pre-commit? 

cc @apsdehal @ebsmothers

Reply:
I think I missed this one. I actually had to fix docs/header.txt because the latest version of the hook automatically prepends `#` to the license. My local state became a mess after this so I had to clean it up properly.